opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/point.py

```diff
@@ -158,9 +182,34 @@ class ResourceMetrics:
     scope_metrics: Sequence[ScopeMetrics]
     schema_url: str
 
+    def to_json(self) -> str:
+        return dumps(
+            {
+                "resource": {
```

Comment:
We should either replicate what we are doing for `resource` in [ReadableSpan](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py#L495) or abstract the logic to a `Resource.to_json()`.

Reply:
Added `to_json` method

Reply:
Could you create an issue to change tracing to use `Resource.to_json()`, in a way that's not breaking?

Reply:
Opened #2747